src/strings/PowerFxResources.en-US.resx

```diff
@@ -5009,6 +5009,14 @@
     <value>This feature is deprecated and is no longer supported.</value>
     <comment>An error message for deprecated features.</comment>
   </data>
+  <data name="WarnDeprecatedFunction" xml:space="preserve">
+    <value>In namespace {1}, function {0} is deprecated.</value>
```

Comment:
>namespace {1}, function {0}  [](http://example.com/codeflow?start=14&length=28)

nit:
I think we can just say  "{0}  is deprecated."   where 0 includes namespace - if present. 
rather than call out namespace separately .